bin-src/mm.coffee

```diff
@@ -54,10 +55,14 @@ createMigration = (opts) ->
     exit "Migration ID is required"
   createMigrator().create cwd(), id, exit, opts.coffee
 
-exit = (err) ->
-  if err
-    console.error "Error: " + err
+exit = (msg, err) ->
+  if debug
+    console.error err.stack
```

Comment:
This must also check for err presence. Because the method is also called to exit successfully